### PR TITLE
Feature profile

### PR DESCRIPTION
**Pull request by katerynablavt**
Here are the changes to change profile usecase

### CODE REVIEW

**Review by xingjianll — CHANGES_REQUESTED** [review]
The package names should be all lowercase.

**Review by xingjianll — APPROVED** [review]
Approved.

File: src/main/java/entities/CommonUser.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class CommonUser implements User
     public CommonUser(int uid, String name, int status)
     {
         this.uid = uid;
-        userProfile = new UserProfile();
+        userProfile = new UserProfile(uid,name);
```
Comment (brendenmcfarlane) [inline src/main/java/entities/CommonUser.java:17]: if every user is storing a profile, it is redundant to store the UID in the profile

File: src/main/java/frameworks_and_drivers/ChangeProfileDB.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package frameworks_and_drivers;
+
+import entities.UserProfile;
+import usecases.profile_changes.ChangeProfileDsInputModel;
+import usecases.profile_changes.ChangeProfileGateWayDB;
+import usecases.profile_changes.ChangeProfileInputModel;
+
+//here i will use ChangeProfileDsInputModel and have access to txt files
+
+public class ChangeProfileDB implements ChangeProfileGateWayDB {
```
Comment (brendenmcfarlane) [inline src/main/java/frameworks_and_drivers/ChangeProfileDB.java:10]: I would add documentation so Joe can implement these

**Review by thmcheung — APPROVED** [review]
When you write good code, you don't need comments - David

**Review by davidlyj — APPROVED** [review]
In ChangeProfileOutputAdapter.java, line 28, the comment is not aligned properly, but the code will still work, just a minor issue. Looks okay to be merged.

**Review by brendenmcfarlane — APPROVED** [review]
I only looked over you files since JOes and Kevins will be overwriting the other files. Ill approve this but add some documentation for when Joe implements the database

**Pull request merged**